config/sources/families/mt7623.conf

```diff
@@ -65,5 +65,8 @@ family_tweaks() {
 
 family_tweaks_bsp() {
```

Comment:
Entirely remove this function? mmc-utils comes from upstream since long time.

Reply:
Yep, removed